scripts.d/s-salome-tar

```diff
@@ -133,6 +133,12 @@ function s-salome-tar-install()
         done
     fi
 
+    #  Do clean-up HELLO and PYHELLO for the release
```

Comment:
The comment should be:
`# Do remove HELLO and PYHELLO from the UI (no icon`)
and a space after this line to separate from the code.